routes/items.js

```diff
@@ -9,9 +9,9 @@ router.get( '/', ( request, response ) => {
   const { Item } = request.app.get( 'models' )
 
   const { user, query } = request
-
   findAllItems( Item, user, query )
-    .then( respondWithItems( user, data => response.render( 'items/index', data ) ) )
+    .then( respondWithItems( user, data =>{
```

Comment:
I'm not seeing why this line changed?  Please return to previous state, since this change is not needed for the current feature